manifest.webapp

```diff
@@ -81,6 +77,11 @@
       "verbs": ["POST"],
       "selector": "worker",
       "values": ["service"]
+    },
+    "bank.operations": {
+      "description": "Used to manage your bank operations",
+      "type": "io.cozy.bank.operations",
+      "verbs": ["GET", "POST", "PUT", "DELETE"]
```

Comment:
You can also use `"verbs": ["ALL"]` https://docs.cozy.io/en/cozy-stack/permissions/#verbs